Changelog.md

```diff
@@ -1,5 +1,8 @@
 ### 0.4.5 (unreleased)
 
+Bugfixes:
+ * Type checker: now more eagerly searches for a commont type of tuple elements of mixed types
```

Comment:
OK, though it's a partial bug fix to me.  The common type should be even more generous.